README.md

```diff
@@ -70,7 +70,7 @@ func main() {
 	// Using the SDK's default configuration, loading additional config
 	// and credentials values from the environment variables, shared
 	// credentials, and shared configuration files
-	cfg, err := external.LoadDefaultAWSConfig()
+	cfg, err := external.LoadDefaultConfig()
```

Comment:
external here needs to be renamed to config too

Reply:
Oh this example is stale. Needs update/removal

Reply:
thanks missed that one